### PR TITLE
Measure publish time E2E tests.

### PR DESCRIPTION
**Pull request by wcs1only**
# Description

Measure and log publish time in for pubsub grpc test.

Output as follows:
```
2021/09/07 19:44:27 Connecting to dapr using url 127.0.0.1:50001
2021/09/07 19:44:27 Dial elapsed: 510.2µs
2021/09/07 19:44:27 NewDaprClient elapsed: 0s
2021/09/07 19:44:27 PublishEvent elapsed: 837.6µs
2021/09/07 19:44:27 Publish succeeded
2021/09/07 19:44:27 performPublish() called
2021/09/07 19:44:27     commandBody.ContentType=application/json
2021/09/07 19:44:27     commandBody.Topic=pubsub-raw-topic-grpc
2021/09/07 19:44:27     commandBody.Data=message-grpc-103
2021/09/07 19:44:27     commandBody.Protocol=grpc
2021/09/07 19:44:27     commandBody.Metadata=map[rawPayload:true]
```

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/apps/pubsub-publisher/app.go [hunk 1]
```diff
@@ -164,14 +164,24 @@ func performPublishGRPC(topic string, jsonValue []byte, contentType string, meta
...
-	conn, err := grpc.Dial(url, grpc.WithInsecure())
+	start := time.Now()
+	conn, err := grpc.Dial(url, grpc.WithInsecure(), grpc.WithBlock())
 	if err != nil {
 		log.Printf("Could not connect to dapr: %s", err.Error())
 		return http.StatusInternalServerError, err
 	}
 	defer conn.Close()
+	t := time.Now()
+	elapsed := t.Sub(start)
```
Comment (pkedy) [inline tests/apps/pubsub-publisher/app.go:175]: Nit: `elapsed := time.Since(start)`

File: tests/apps/pubsub-publisher/app.go [hunk 2]
```diff
@@ -182,6 +192,11 @@ func performPublishGRPC(topic string, jsonValue []byte, contentType string, meta
 	}
 	_, err = client.PublishEvent(context.Background(), req)
 
+	t = time.Now()
+	elapsed = t.Sub(start)
```
Comment (pkedy) [inline tests/apps/pubsub-publisher/app.go:196]: Nit: `elapsed := time.Since(start)`

**Review by pkedy — APPROVED** [review]

**Review by pkedy — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

**Pull request merged**